### ISSUE TITLE
Single row heatmap does not show

### ISSUE THREAD

**Issue by halhen** [ropensci/iheatmapr#78]
Version 0.4.12:

Plotting a single-row main heatmap results in an all-white plot. I -suspect- that the data is plotted, but with something like a 0-pixel height.

Reproducible example:

## 2 rows works as you expect
```
library(iheatmapr)
B <- matrix(1:6, ncol = 3)
main_heatmap(B)
```
![image](https://user-images.githubusercontent.com/71583/88169355-27f8f880-cc1c-11ea-9c34-fd75bbfa59ec.png)

## 1 row does not show
```
A <- matrix(1:3, ncol = 3)
main_heatmap(A)
```
![image](https://user-images.githubusercontent.com/71583/88169437-39da9b80-cc1c-11ea-9b10-a81968ef3fe4.png)

## Workaround /-ish: include a NA-only dummy row
```
A_dummy <- matrix(c(1:3, NA, NA, NA), ncol = 3, byrow = TRUE)
main_heatmap(A_dummy)
```
![image](https://user-images.githubusercontent.com/71583/88169504-5676d380-cc1c-11ea-90f7-a7a0349f339d.png)

## Workaround /-ish 2: duplicate the single row
```
A2 <- rbind(A, A)
main_heatmap(A2)
```

![image](https://user-images.githubusercontent.com/71583/88170125-72c74000-cc1d-11ea-8a08-3c575db6ccf1.png)

**Comment by AliciaSchep** [comment]
This issue should be fixed in version 0.5.0 and above.

**Comment by halhen** [comment]
Ah, sweet. Thanks, and sorry for not checking myself. 🌷
